app/src/main/java/org/p2p/wallet/debug/settings/DebugSettingsPresenter.kt

```diff
@@ -102,16 +101,16 @@ class DebugSettingsPresenter(
 
     private fun getCiInfo(): List<SettingsRow> {
         val ciValues = buildString {
-            createApiKeyRecord("amplitudeKey", BuildConfig.amplitudeKey)
-            createApiKeyRecord("intercomApiKey", BuildConfig.intercomApiKey)
-            createApiKeyRecord("intercomAppId", BuildConfig.intercomAppId)
-            createApiKeyRecord("moonpayKey", BuildConfig.moonpayKey)
-            createApiKeyRecord("moonpaySanbdoxKey", BuildConfig.moonpaySandboxKey)
-            createApiKeyRecord("rpcPoolApiKey", BuildConfig.rpcPoolApiKey)
+            createApiKeyRecord("amplitudeKey", org.p2p.wallet.BuildConfig.amplitudeKey)
```

Comment:
aliases for imports
import org.p2p.core.BuildConfig as CoreBuildConfig
import org.p2p.wallet.BuildConfig as AppBuildConfig